buildtools/tinygo/wasi-libc.Dockerfile

```diff
@@ -5,11 +5,8 @@ FROM ghcr.io/corazawaf/coraza-proxy-wasm/buildtools-wasi-sdk:main
 
 RUN apt-get install -y git
 
-# Includes
-# https://github.com/tinygo-org/tinygo/pull/3245
-# https://github.com/tinygo-org/tinygo/pull/3252
-RUN git clone https://github.com/anuraaga/tinygo --branch coraza-fork
+RUN git clone https://github.com/tinygo-org/tinygo
 WORKDIR /tinygo
-RUN git fetch origin && git reset --hard f20979998e57fc5128fbca823b3897b66467abe1
```

Comment:
Guessing unintentional but let's upgrade TinyGo in a separate PR

Reply:
yeah makes sense.